src/main/cc/wfa/virtual_people/core/model/utils/distributed_consistent_hashing_test.cc

```diff
@@ -166,45 +163,39 @@ TEST(DistributedConsistentHashingTest, TestZeroAfterNormalization) {
 }
 
 TEST(DistributedConsistentHashingTest,
-     TestOutputDistribution_NonConsecutiveChoiceId) {
+     TestOutputDistributionNonConsecutiveChoiceId) {
   // Distribution:
   // choice_id probability
   // 0         0.4
   // 2         0.2
   // 4         0.2
   // 6         0.2
-  auto distribution = absl::make_unique<std::vector<DistributionChoice>>();
-  distribution->emplace_back(0, 0.4);
-  distribution->emplace_back(2, 0.2);
-  distribution->emplace_back(4, 0.2);
-  distribution->emplace_back(6, 0.2);
+  std::vector<DistributionChoice> distribution({
+    DistributionChoice({0, 0.4}),
+    DistributionChoice({2, 0.2}),
+    DistributionChoice({4, 0.2}),
+    DistributionChoice({6, 0.2})
+  });
   ASSERT_OK_AND_ASSIGN(
       std::unique_ptr<DistributedConsistentHashing> hashing,
       DistributedConsistentHashing::Build(std::move(distribution)));
-  absl::flat_hash_map<int32_t, int> output_counts = {
-      {0, 0},
-      {2, 0},
-      {4, 0},
-      {6, 0}};
+  absl::flat_hash_map<int32_t, double> output_counts;
   for (int seed = 0; seed < kSeedNumber; ++seed) {
     int32_t output = hashing->Hash(std::to_string(seed));
-    // The output should always be one of [0, 2, 4, 6].
-    EXPECT_TRUE(output_counts.find(output) != output_counts.end());
+    if (output_counts.find(output) == output_counts.end()) {
```

Comment:
I think this whole if-statement is unnecessary. When you do `output_counts[some_key]`, it will insert a new default value at that key if the key does not yet exist. For doubles, this is 0.

Reply:
Done.